.github/reusable-lint.yml

```
@@ -0,0 +1,19 @@
+name: Reusable Lint Workflow
+
+on: workflow_call
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+      # Check the package.json is correctly formed
+      - run: npx package-check
```

I couldn't find any info on this; could you link the package?

---

Looks like it should be this instead and some npx/npm weirdness is happening here:
```suggestion
      - run: npx @skypack/package-check
```

---

> We found out about it from you

This used the template, which probably used an older version of solid-client... I guess I didn't notice it before.

---

Have switch to the suggestion above.